docs/rules/no-sequences.md

```diff
@@ -61,6 +63,8 @@ switch ((val = foo(), val)) {}
 while ((val = foo(), val < 42));
 
 with ((doSomething(), val)) {}
+
+const foo = (val) => ((console.log('bar'), val));
```

Comment:
The examples are good! Could you please also add this same example to the [allowInParentheses](https://eslint.org/docs/rules/no-sequences#allowinparentheses) section? It would be in the existing "incorrect" examples codeblock for `{ "allowInParentheses": false }`.